### DISCUSSION TITLE
Responsive control for number of posts in Loop Grid

### DISCUSSION THREAD

**Discussion by pumadog** [elementor/elementor#24843] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

In the Loop Grid I miss an option to display different numbers of posts for different viewports.

### Describe the solution you'd like.

Please add responsive controls for the number of loop items to display.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by wagnerch1981** [comment]
Yes, this function should definitely be added. Otherwise you have to duplicate the loops unnecessarily every time for every screen size for a clean resonsive design
